### PR TITLE
MAGECLOUD-3469 indicate add-ons in starter and pro

### LINKED ISSUE #4202: Starter B2B module Feedback on page: /guides/v2.3/cloud/architecture/cloud-architecture.html
The B2B module is not included with Starter.  

It is available for use with Starter for an additional fee.

### PR DESCRIPTION
**Pull request by hguthrie**
Updated the Cloud architecture matrix to show features available for an additional fee, such as B2B for starter.

Fixes #4202 

whatsnew
Added a row for Add-ons to the [Cloud Architecture table](https://devdocs.magento.com/guides/v2.2/cloud/architecture/cloud-architecture.html).

### CODE REVIEW

File: guides/v2.1/cloud/architecture/cloud-architecture.md [hunk 1]
```diff
@@ -85,8 +84,18 @@ For comparison, each plan includes the following infrastructure features and sup
       <td>Dedicated technical account management for the initial launch period, starting with your subscription until your initial site launch</td>
       <td>Dedicated technical account management for the initial launch period, starting with your subscription until your initial site launch</td>
     </tr>
+    <tr>
+      <td>Add-ons*</td>
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/architecture/cloud-architecture.md]: Should there also be an entry for B2B in the Pro column?

Comment (hguthrie) [reply guides/v2.1/cloud/architecture/cloud-architecture.md]: I thought Pro comes with B2B. Is it an add-on at an extra cost?

Comment (jeff-matthews) [reply guides/v2.1/cloud/architecture/cloud-architecture.md]: Oh, I think you're right.

**Comment by hguthrie** [comment]
running tests

**Pull request merged**